subprojects/docs/src/docs/userguide/jacocoPlugin.xml

```diff
@@ -236,6 +241,16 @@
                 </td>
                 <td>Generates code coverage report for the test task.</td>
             </tr>
+            <tr>
+                <td>
+                    <literal>jacocoTestCheck</literal>
```

Comment:
Nitpick: this task name doesn't represent what it's for IMO. Perhaps `jacocoVerificationCheck` or `jacocoEnforceThresholds` or something.